src/Youtube.js

```diff
@@ -298,26 +298,28 @@ THE SOFTWARE. */
 
     onPlayerError: function(e) {
       this.errorNumber = e.data;
+      this.trigger('pause');
       this.trigger('error');
-
-      this.ytPlayer.stopVideo();
     },
 
     error: function() {
       switch (this.errorNumber) {
```

Comment:
I see. Well, I don't see the point of aligning errors as you might want to handle Youtube errors differently. What about adding a fixed amount (e.g.`1000`) to the `errorNumber`? So devs know that codes between `1000-2000` are Youtube specific errors.